openfasoc/generators/gdsfactory-gen/glayout/primitives/fet.py

```diff
@@ -547,6 +582,39 @@ def pmos(
             horizontal_glayer=substrate_tap_layers[0],
             vertical_glayer=substrate_tap_layers[1],
         )
-    return rename_ports_by_orientation(pfet).flatten()
+    component =  rename_ports_by_orientation(pfet).flatten()
+
+    # add spice netlist
+    num_dummies = 0
```

Comment:
Up to you how you want to do this, but notice how the nmos and pmos netlist process is almost identical. You can actually make this one function (up to you tho)